README.md

```diff
@@ -39,10 +38,25 @@ layer.addTo(map);
 map.fitBounds(layer.getBounds());
 ```
 
-# advanced usage
+The following options are supported:
+
+| **Option**                   | **Data Type** | **Default value**                          | **Description**                                                                                                                                                                                     |
+|------------------------------|---------------|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| baseUrl                      | string        | `href` of the `self` link in the STAC data | The base URL for relative links. Should be provided if the STAC data has no self link and links are not absolute. |
+| buildUrlTemplate             | function      | undefined                                  | For server-side rendering of imagery. See the chapter [using a tiler](#using-a-tiler) and [buildTileUrlTemplate](#buildTileUrlTemplate) for details. |
+| displayPreview               | boolean       | false                                      | Allow to display the asset with role `thumbnail` or the link with relation type `preview`. The previews are usually not covering the full extents and as such may be placed incorrectly on the map. |
+| displayOverview              | boolean       | true                                       | Allow to display the asset with role `overview` or `visual`. |
+| debugLevel                   | integer       | 0                                          | The higher the value the more debugging messages will be logged to the console. `0` to disable logging. |
+| latLngBounds / bounds / bbox | array<number> | undefined                                  | For STAC Assets only: The bounding box of the asset |
```

Comment:
```suggestion
| latLngBounds / bounds / bbox | array\<number> | undefined                                  | For STAC Assets only: The bounding box of the asset |
```

And we likely need to tell what the array should contain for all these options...

Reply:
Maybe we can update dataType to 

[latLngBounds](https://leafletjs.com/reference.html#latlngbounds) | [bounds](https://leafletjs.com/reference.html#bounds) | [West, South, East, North]

or 

[latLngBounds](https://leafletjs.com/reference.html#latlngbounds) | [bounds](https://leafletjs.com/reference.html#bounds) | [MinLongitude, MinLatitude, MaxLongitude, MaxLatitude]

Reply:
Done.